### PR TITLE
Haokun yang123/gallery home page

### LINKED ISSUE #59: Implement Gallery "Home" Page
image

- Use Title.jsx for the title
- Use Subtitle.jsx for the smaller titles.
- Food Tabling use regular Next Image (Not Picture.jsx)
- For Quarterly and Past Events, use Hoverable.jsx
- Use Tailwind Grid

Tabling can done without mapping since it's only three images.
For Quarterly Events, you will need to create a data file containing an object. Each object contains a title, link, and photo. You will need to map for fall, winter and spring.

Pictures can be found in the public folder.

### PR DESCRIPTION
**Pull request by HaokunYang123**
image

### CODE REVIEW

**Comment by HaokunYang123** [comment]
The Subtitle somehow isnt centered so I changed it an re PR

File: src/app/gallery/page.js [hunk 1]
```diff
@@ -1,5 +1,60 @@
...
-  return <div>gallery</div>;
+  return (
+    <div className="bg-green-100 min-h-screen p-7 w-full flex flex-col items-center">
+      <div className="flex justify-center mb-10">
+        <Title text="Gallery" food="chip" color="biscuits-grey-100" />
+      </div>
+
+      <div className="mb-14">
+        <div className="mb-10 flex justify-center">
+          <Subtitle text="Food Tabling" color="biscuits-green-200" />
```
Comment (seanquiambao) [inline src/app/gallery/page.js]:
```suggestion
          <Subtitle text="Food Tabling" color="text-biscuits-green-200" />
```

repeat for other subtitles

File: src/app/gallery/page.js [hunk 2]
```diff
@@ -1,5 +1,60 @@
...
+          ))}
+        </div>
+      </div>
+
+      <div className="mb-14">
+        <div className="mb-10 flex justify-center">
+          <Subtitle text="Past Events" color="green-200" />
+        </div>
+        <div className="text-textColor">
+          <Hoverable link="/PastEvents" image={PastEvent} text="Past Events" />
```
Comment (seanquiambao) [inline src/app/gallery/page.js]:
```suggestion
          <Hoverable link="/past" image={PastEvent} text="Past Events" />
```

**Review by seanquiambao — CHANGES_REQUESTED** [review]

**Comment by seanquiambao** [comment]
link issue

File: src/app/gallery/page.js [hunk 3]
```diff
@@ -1,5 +1,60 @@
...
+import Tabling2 from "@/public/images/tabling/tabling_2.webp";
+import Tabling3 from "@/public/images/tabling/tabling_3.webp";
+import PastEvent from "@/public/images/past/past_3.webp";
+
 const Page = () => {
-  return <div>gallery</div>;
+  return (
+    <div className="bg-green-100 min-h-screen p-7 w-full flex flex-col items-center">
+      <div className="flex justify-center mb-10">
+        <Title text="Gallery" food="chip" color="biscuits-grey-100" />
```
Comment (seanquiambao) [inline src/app/gallery/page.js]:
```suggestion
        <Title text="Gallery" food="chip" />
```

**Review by seanquiambao — APPROVED** [review]

**Review by seanquiambao — APPROVED** [review]

**Pull request merged**